src/VisualStudio/Core/Impl/Microsoft.VisualStudio.LanguageServices.Implementation.csproj

```diff
@@ -39,6 +39,7 @@
     <Reference Include="WindowsFormsIntegration" />
     <EmbeddedResource Update="SolutionExplorerShim.resx" GenerateSource="true" />
     <PackageReference Include="Microsoft.CSharp" />
+    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem" />
```

Comment:
@lifengl would it be better to use the SDK package here, to get CPS's analyzers etc?